RevenueCat.podspec

```diff
@@ -31,5 +31,5 @@ Pod::Spec.new do |s|
   s.source_files = 'Sources/**/*.swift'
   s.exclude_files = 'Sources/LocalReceiptParsing/ReceiptParser-only-files/**'
   
-  
+  s.resource_bundles = {'RevenueCat' => ['Sources/PrivacyInfo.xcprivacy']}
```

Comment:
Hmm I was wondering if we also need this for `RevenueCatUI` but I think it shouldn't be needed, since `RevenueCat` is also a dependency for `RevenueCatUI` right?

Reply:
Yeah, that's my understanding. Having it only in `RevenueCat` would also match our SPM setup. 
In any case even if we did need to add it we could do it as a separate PR, it doesn't really take value away from merging this one